docs/advanced.rst

```diff
@@ -1117,12 +1117,13 @@ SoC/firmware and storage medium.
 
 .. _sec-emmc-boot:
 
-Update eMMC Boot Partitions
-~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Update eMMC Bootloader Partitions
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-RAUC supports updating eMMC boot partitions (see the JEDEC standard JESD84-B51_
-for details), one of which gets enabled atomically via configuration registers
-in the eMMC (*ext_csd registers*).
+RAUC supports updating eMMC bootloader partitions (see the JEDEC standard
```

Comment:
The standard refers to them as `boot partitions`. Thus we might have to be careful here to differentiate between the storage type `eMMC boot partition` and the usage `bootloader partition`.